com.paledega.alpinelinux.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.paledega.alpinelinux
```

Comment:
`paledega.com` doesn't resolve. Do you really own the domain?

Reply:
No. I can change with com.github.xxx